src/iiiftilesource.js

```diff
@@ -536,16 +559,32 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
         ];
         var profileLevel = Array.isArray(options.profile) ? options.profile[0] : options.profile;
         var isLevel0 = (level0Profiles.indexOf(profileLevel) !== -1);
-        var hasCanoncicalSizeFeature = false;
+        return isLevel0;
+    }
+
+
+    /**
+     * Determine whether arbitrary tile requests can be made against a service with the given profile
+     * @function
+     * @param {Object} options
+     * @param {Array|String} options.profile
+     * @param {Number} options.version
+     * @param {String[]} options.extraFeatures
+     * @returns {Boolean}
+     */
+    function canBeTiled ( options ) {
+        var isLevel0 = checkLevel0( options );
+        var hasCanonicalSizeFeature = false;
         if ( options.version === 2 && options.profile.length > 1 && options.profile[1].supports ) {
-            hasCanoncicalSizeFeature = options.profile[1].supports.indexOf( "sizeByW" ) !== -1;
+            hasCanonicalSizeFeature = options.profile[1].supports.indexOf( "sizeByW" ) !== -1;
         }
         if ( options.version === 3 && options.extraFeatures ) {
-            hasCanoncicalSizeFeature = options.extraFeatures.indexOf( "sizeByWh" ) !== -1;
+            hasCanonicalSizeFeature = options.extraFeatures.indexOf( "sizeByWh" ) !== -1;
         }
-        return !isLevel0 || hasCanoncicalSizeFeature;
+        return !isLevel0 || hasCanonicalSizeFeature;
     }
 
+
```

Comment:
Is this extra blank line needed? (Not sure about style conventions of the project, but wondering if it was added by accident).